lively.ide/components/editor.js

```diff
@@ -192,7 +192,7 @@ export class InteractiveComponentDescriptor extends ComponentDescriptor {
 
   getDependants (immediate = false) {
     return $world.withAllSubmorphsSelect(m =>
-      m.master?.uses(this.stylePolicy, immediate)
+      m.master?.uses?.(this.stylePolicy, immediate)
```

Comment:
How comes that if `master: false` the first save-navigation operator does not suffice?

Reply:
This was also news for me: apparently the conditional member expression only cancels further lookups when a requested member is `undefined`. Since `false` is not `undefined` the lookup proceeds and attempts to call the `uses()` function on `false` which fails.